libcloudforensics/gcp.py

```diff
@@ -331,19 +341,21 @@ def GetOrCreateAnalysisVm(
     Args:
       vm_name (str): Name of the virtual machine.
       boot_disk_size (int): The size of the analysis VM boot disk (in GB).
-      cpu_cores (int): Number of CPU cores for the virtual machine.
-      image_project (str): Name of the project where the analysis VM image is
-        hosted.
-      image_family (str): Name of the image to use to create the analysis VM.
-      packages (list[str]): List of packages to install in the VM.
+      cpu_cores (int): Optional. Number of CPU cores for the virtual machine.
+      image_project (str): Optional. Name of the project where the analysis VM
```

Comment:
It would be nice to describe the behavior of the function if an optional parameter is not provided.